sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -3,479 +3,563 @@
+  readonly enqueuedTimeUtc?: Date;
+  readonly expiresAtUtc?: Date;
+  get isSettled(): boolean;
+  label?: string;
+  lockedUntilUtc?: Date;
+  readonly lockToken?: string;
+  messageId?: string | number | Buffer;
+  partitionKey?: string;
+  replyTo?: string;
+  replyToSessionId?: string;
+  scheduledEnqueueTimeUtc?: Date;
+  readonly sequenceNumber?: Long;
+  sessionId?: string;
+  timeToLive?: number;
+  to?: string;
+  userProperties?: {
+    [key: string]: any;
+  };
+  viaPartitionKey?: string;
+}
+
+// @public (undocumented)
+export interface ServiceBusReceiverClient {
+  new (queueAuth: QueueAuth, receiveMode: "peekLock"): ClientTypeT<
+    "peekLock",
+    "queue",
+    "nosessions"
+  >;
+  new (queueAuth: QueueAuth, receiveMode: "receiveAndDelete"): ClientTypeT<
+    "receiveAndDelete",
+    "queue",
+    "nosessions"
+  >;
+  new (queueAuth: QueueAuth, receiveMode: "peekLock", session: Session): ClientTypeT<
+    "peekLock",
+    "queue",
+    "sessions"
+  >;
+  new (queueAuths: QueueAuth, receiveMode: "receiveAndDelete", session: Session): ClientTypeT<
+    "receiveAndDelete",
+    "queue",
+    "sessions"
+  >;
+  new (subscriptionAuth: SubscriptionAuth, receiveMode: "peekLock"): ClientTypeT<
+    "peekLock",
+    "subscription",
+    "nosessions"
+  >;
+  new (subscriptionAuth: SubscriptionAuth, receiveMode: "receiveAndDelete"): ClientTypeT<
+    "receiveAndDelete",
+    "subscription",
+    "nosessions"
+  >;
+  new (subscriptionAuth: SubscriptionAuth, receiveMode: "peekLock", session: Session): ClientTypeT<
+    "peekLock",
+    "subscription",
+    "sessions"
+  >;
+  new (
+    subscriptionAuth: SubscriptionAuth,
+    receiveMode: "receiveAndDelete",
+    session: Session
+  ): ClientTypeT<"receiveAndDelete", "subscription", "sessions">;
 }
 
 // @public
-export interface ServiceBusClientOptions {
-    dataTransformer?: DataTransformer;
-    webSocketOptions?: WebSocketOptions;
+export const ServiceBusReceiverClient: ServiceBusReceiverClient;
+
+// @public (undocumented)
+export class ServiceBusSenderClient {
+  constructor(entityConnectionString: string, options?: ServiceBusClientOptions);
+  constructor(
+    serviceBusConnectionString: string,
+    entityName: string,
+    options?: ServiceBusClientOptions
+  );
+  constructor(
+    host: string,
+    entityName: string,
+    credential: TokenCredential,
+    options?: ServiceBusClientOptions
+  );
+  // (undocumented)
+  cancelScheduledMessage(sequenceNumber: Long): Promise<void>;
+  // (undocumented)
+  cancelScheduledMessages(sequenceNumbers: Long[]): Promise<void>;
+  // (undocumented)
+  close(): Promise<void>;
+  // (undocumented)
+  _entityPath: string;
+  // (undocumented)
+  scheduleMessage(scheduledEnqueueTimeUtc: Date, message: SendableMessageInfo): Promise<Long>;
+  // (undocumented)
+  scheduleMessages(scheduledEnqueueTimeUtc: Date, messages: SendableMessageInfo[]): Promise<Long[]>;
+  // (undocumented)
+  send(message: SendableMessageInfo): Promise<void>;
```

Comment:
For track-2 we support cancellation tokens. Are we planning to add parameter `AbortSignalLike`?

Reply:
Added in operationOptions parameters which contain the abort signal (and tracing parameters).